packages/components/src/lib/draw/index.ts

```diff
@@ -219,9 +219,11 @@ export function drawCursorLines({
     context.beginPath();
     context.strokeStyle = cursorLineColor;
     context.lineWidth = 1;
+    context.setLineDash([5, 5]); // setting the dashed line pattern
     context.moveTo(point.x, 0);
     context.lineTo(point.x, frame.height);
     context.stroke();
+    context.setLineDash([]); // resetting the dashed line pattern so it doesn't affect other lines
```

Comment:
Note that it's also possible to do `context.save()` + `context.restore()`. But I wasn't very consistent with these when I wrote this module.

Reply:
Good to know. I'm not very used to this, used GPT4 to help with this part.